### PR TITLE
feat/commands executed and songs played counter activated

### PR DESCRIPTION
**Pull request by BySuspect**

### CODE REVIEW

File: djs-bot/lib/clients/Erela.js [hunk 1]
```diff
@@ -266,6 +266,7 @@ module.exports = (client) => {
 
                   // !TODO: integrate events with socket
 		.on("trackStart", async (player, track) => {
+			client.songsPlayed++;
```
Comment (Neko-Life) [inline djs-bot/lib/clients/Erela.js]: add this to cosmicord client too

**Review by Neko-Life — CHANGES_REQUESTED** [review]
pretty good addition, only one thing

File: djs-bot/lib/clients/Erela.js (hunk 1 above)
Comment (BySuspect) [reply djs-bot/lib/clients/Erela.js]: It was already integrated into the API and since the values were always 0, it showed 0 on the dashboard.

Comment (Neko-Life) [reply djs-bot/lib/clients/Erela.js]: what i mean is [cosmicord's trackStart event](https://github.com/wtfnotavailable/Discord-MusicBot/blob/develop/djs-bot/lib/clients/Cosmicord.js#L224) need to be handled too

What you should do exactly is remove the line here and put it in [handleTrackStart](https://github.com/wtfnotavailable/Discord-MusicBot/blob/develop/djs-bot/lib/MusicEvents.js#L88) instead

**Review by Neko-Life — APPROVED** [review]

**Pull request merged**